test/built-ins/BigInt/asIntN/bits-toindex.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: BigInt.asIntN type coercion for bits parameter
+info: >
+  BigInt.asIntN ( bits, bigint )
+
+  1. Let bits be ? ToIndex(bits).
+features: [Symbol]
```

Comment:
this should add the `arrow-function` feature as well, or at least use regular functions instead.